server/models/Request.js

```diff
@@ -0,0 +1,24 @@
+import mongoose from "mongoose";
+
+const { Schema, model } = mongoose;
+
+const requestSchema = new mongoose.Schema(
+  {
+    customer: { type: Schema.Types.ObjectId, ref: "User", required: true },
+    provider: { type: Schema.Types.ObjectId, ref: "User", required: true },
+    start: { type: Date, required: true },
+    end: { type: Date, required: true },
+    accepted: { type: Boolean, default: false },
+    declined: { type: Boolean, default: false },
+    paid: { type: Boolean, default: false },
+    serviceType: {
+      type: String,
+      enum: ["Boarding", "House Sitting", "Day Care", "Walking"],
+    },
```

Comment:
This is a nice addition to the basic schema, also nice use of the `enum` validator! 🙌🏼

One thing I would suggest would be to normalize values to either `lowercase/uppercase` to be more flexible in terms of casing ([docs](https://mongoosejs.com/docs/schematypes.html#string-validators)).